modules/svg/doc_classes/ResourceImporterLottie.xml

```diff
@@ -0,0 +1,129 @@
+			Note that RGTC compression affects the resulting normal map image. You will have to adjust custom shaders that use the normal map's blue channel to take this into account. Built-in material shaders already ignore the blue channel in a normal map (regardless of the actual normal map's contents).
+		</member>
+		<member name="detect_3d/compress_to" type="int" setter="" getter="" default="1">
+			This changes the [member compress/mode] option that is used when a texture is detected as being used in 3D.
+			Changing this import option only has an effect if a texture is detected as being used in 3D. Changing this to [b]Disabled[/b] then reimporting will not change the existing compress mode on a texture (if it's detected to be used in 3D), but choosing [b]VRAM Compressed[/b] or [b]Basis Universal[/b] will.
+		</member>
+		<member name="editor/convert_colors_with_editor_theme" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], converts the imported image's colors to match [member EditorSettings.interface/theme/icon_and_font_color]. This assumes the image uses the exact same colors as [url=$DOCS_URL/contributing/development/editor/creating_icons.html]Godot's own color palette for editor icons[/url], with the source file designed for a dark editor theme. This should be enabled for editor plugin icons and custom class icons, but should be left disabled otherwise.
+			[b]Note:[/b] Only available for SVG images.
+		</member>
+		<member name="editor/scale_with_editor_scale" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], scales the imported image to match [member EditorSettings.interface/editor/custom_display_scale]. This should be enabled for editor plugin icons and custom class icons, but should be left disabled otherwise.
+			[b]Note:[/b] Only available for SVG images.
+		</member>
+		<member name="lottie/begin" type="float" setter="" getter="" default="0">
+			The start of the Lottie animation in the range [code]0.0[/code] to [code]1.0[/code].
+		</member>
+		<member name="lottie/columns" type="int" setter="" getter="" default="0">
+			The number of columns of the sprite sheet. If it is 0, the number of columns will be automatically calculated to be close to the number of rows.
+		</member>
+		<member name="lottie/end" type="float" setter="" getter="" default="1">
+			The end of the Lottie animation in the range [code]0.0[/code] to [code]1.0[/code]. Less than [member lottie/begin] will be considered as [member lottie/begin].
```

Comment:
You can also say as follows, assuming I understand it right. I wouldn't lose sleep over it, though.
```suggestion
			The end of the Lottie animation in the range [code]0.0[/code] to [code]1.0[/code]. A value lower than [member lottie/begin] will be clamped to [member lottie/begin].
```
or
```suggestion
			The end of the Lottie animation in the range [code]0.0[/code] to [code]1.0[/code]. This value will be clamped to be at least [member lottie/begin].
```
Although, ideally, this value should be clamped automatically in the importer itself, but we may have no system for this (?).